flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -686,27 +680,38 @@ private void shutdownAsyncThreads() throws Exception {
 		}
 	}
 
+	private Exception suppressThrowable(ThrowingRunnable<?> runnable, Exception suppressedException) {
```

Comment:
1. method name `suppressThrowable` doesn't say that it runs something. `runSuppressed`?
1. argument name `suppressedException` is a bit misleading because it is `e` that will be suppressed. `originalException`?
1. `@Nullable` for the 2nd argument?

Reply:
1 -> how about `runAndSuppressException`?
2, 3 agree.